Add instructions to activate in Sphinx (#4)
pradyunsg/sphinx-inline-tabs: README.md

pip install sphinx-inline-tabs
```

> activate this extension in Sphinx by adding the following to your `conf.py`:

Reviewer: ```suggestion
You'll also want to add the extension to `extensions` in `conf.py`:
```
